mars/chain.json

```diff
@@ -123,6 +132,10 @@
       {
         "address": "https://grpc-mars.nodeist.net:443",
         "provider": "Nodeist"
+      },
+      {
+        "address": "mars-grpc.polkachu.com:18590",
```

Comment:
I assume it was an accident to leave out 'https://' here?

Reply:
It is intentional. See the grpcurl command and result below. My gPRC endpoint only works without http or https. 

I do not how how other folks are able to reverse-proxy it. I have never found a good solution. Instead, my endpoint is a simple DNS lookup plus the "naked" gRPC port. 

```
grpcurl -plaintext mars-grpc.polkachu.com:18590 list
cosmos.auth.v1beta1.Query
cosmos.authz.v1beta1.Query
cosmos.bank.v1beta1.Query
cosmos.base.reflection.v1beta1.ReflectionService
cosmos.base.reflection.v2alpha1.ReflectionService
cosmos.base.tendermint.v1beta1.Service
cosmos.distribution.v1beta1.Query
cosmos.evidence.v1beta1.Query
cosmos.feegrant.v1beta1.Query
cosmos.gov.v1.Query
cosmos.gov.v1beta1.Query
cosmos.params.v1beta1.Query
cosmos.slashing.v1beta1.Query
cosmos.staking.v1beta1.Query
cosmos.tx.v1beta1.Service
cosmos.upgrade.v1beta1.Query
cosmwasm.wasm.v1.Query
grpc.reflection.v1alpha.ServerReflection
ibc.applications.transfer.v1.Query
ibc.core.channel.v1.Query
ibc.core.client.v1.Query
ibc.core.connection.v1.Query
mars.incentives.v1beta1.Query
mars.safety.v1beta1.Query
```